Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -169,22 +169,22 @@ Command keywords and values
    :Type:
          integer
    :Value:
-         Value is an uid of the :php:`sys_language` to localize the record into.
+         Value is an uid of the :sql:`sys_language` to localize the record into.
          Basically a localization of a record is making a copy of the record
-         (possibly excluding certain fields defined with :php:`l10n_mode`) but
-         changing relevant fields to point to the right sys language / original
+         (possibly excluding certain fields defined with :sql:`l10n_mode`) but
+         changing relevant fields to point to the right :sql:`sys_language` / original
          language record.
 
          Requirements for a successful localization is this:
 
          - :code:`[ctrl]` options "languageField" and "transOrigPointerField" must be
```

Comment:
We could have `:php:` for these options too.

Reply:
I already pushed these changes, but I'm not sure why they still appear here as not changed.

Reply:
Not sure either but if it doesn't show up on the latest state, can you push them again?